.github/workflow-examples/automatic_deployment_production.yml

```diff
@@ -0,0 +1,51 @@
+# Deploy Theme to Store when new changes are pushed to master branch
+name: Deploy Theme to Store
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master, main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node: [12.x]
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node }}
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: ${{ matrix.node }}
+
+    - name: npm cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+
+    - name: Install Stencil CLI Dependency
+      run: npm install -g @bigcommerce/stencil-cli
+
+    - name: Install Dependencies
+      run: npm ci
+
+#
+# You must configure store credentials as secrets on your GitHub repo for automatic deployment via GitHub Actions
+#
+
+    - name: Connect to store
+      env:
+        URL: ${{ secrets.STENCIL_STORE_URL_PRODUCTION }}
```

Comment:
Do we want to add a readme file as well in this repo to call out how to use it and how to setup some of these secrets ?

Reply:
I plan to get some public dev docs posted... can do another PR to add a link once we have those